Tests/MetricsTests/TestMetrics.swift

```diff
@@ -135,17 +139,33 @@ internal class TestRecorder: RecorderHandler, Equatable {
 internal class TestTimer: TimerHandler, Equatable {
     let id: String
     let label: String
+    let displayUnit: TimeUnit
     let dimensions: [(String, String)]
 
     let lock = NSLock()
     var values = [(Date, Int64)]()
 
-    init(label: String, dimensions: [(String, String)]) {
+    init(label: String, prefferedDisplayUnit displayUnit: TimeUnit, dimensions: [(String, String)]) {
```

Comment:
typo? `preffered` -> `preferred` 

https://www.grammar.com/preferred-correct-spelling

Reply:
Aw snap. English really got me like that. I'll fix that! Thanks!